app/views/campaigns/borrowing_get_the_facts.html.erb

```diff
@@ -0,0 +1,89 @@
+<div class="l-constrained">
+  <article>
+    <div class="l-campaign__content l-campaign__content--padded-top">
+      <%= heading_tag 'Borrowing - get the facts', class: 'l-campaign__heading', level: 1 %>
```

Comment:
As I see more in the bottom, the campaigns not using locales. Is this page supposed to be on Contento :tm: ?